### PR TITLE
Create validator for biz profile request

### PR DESCRIPTION
**Pull request by ruaraya**
Add validation for business branches in business profile request, including related test cases

### CODE REVIEW

File: src/Spd.Manager.Licence/BizProfileValidation.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using FluentValidation;
+using Microsoft.IdentityModel.Tokens;
```
Comment (peggy-quartech) [inline src/Spd.Manager.Licence/BizProfileValidation.cs:2]: why we need this?

Comment (ruaraya) [reply src/Spd.Manager.Licence/BizProfileValidation.cs:2]: it is used by the method `IsNullOrEmpty()`

File: src/Spd.Manager.Licence/BizProfileValidation.cs [hunk 2]
```diff
@@ -21,6 +22,17 @@ public BizProfileUpdateRequestValidator()
         RuleFor(r => r.SoleProprietorSwlPhoneNumber)
             .NotEmpty()
             .When(r => r.BizTypeCode == BizTypeCode.NonRegisteredSoleProprietor || r.BizTypeCode == BizTypeCode.RegisteredSoleProprietor);
-        //todo: add rule for branches
+        RuleFor(r => r.Branches)
+            .ForEach(r => r
+                .Must(r => r.BranchAddress?.AddressLine1.IsNullOrEmpty() != true)
```
Comment (peggy-quartech) [inline src/Spd.Manager.Licence/BizProfileValidation.cs:27]: do not use it if it is from Token.
Just use string.IsNullOrWhiteSpace.

**Review by peggy-quartech — CHANGES_REQUESTED** [review]

File: src/Spd.Manager.Licence/BizProfileValidation.cs (hunk 2 above)
Comment (ruaraya) [reply src/Spd.Manager.Licence/BizProfileValidation.cs]: Fixed.

**Review by peggy-quartech — APPROVED** [review]

**Pull request merged**